### DISCUSSION TITLE
How do I disable "jj" = "<ESC>" in INSERT mode? i want to type "jj"s sometimes ....

### DISCUSSION THREAD

**Discussion by noamso1** [NvChad/NvChad#2413] [General]
I've tried to add these lines to **lua/custom/mappings.lua**:
```
  i = {
    ["kk"] = { "kkkk" },
    ["jj"] = { "jjjj" },
  },
```
And when i type "kk" indeed it turns into "kkkk" but the "jj" mapping does not work, it escapes to NORMAL mode.

any idea how to get rid of the "jj" binding?

**Comment by noamso1** [comment]
@siduck  maybe you know ?

**Reply by siduck** [reply]
i have no idea what you are trying to do!!

**Reply by noamso1** [reply]
@siduck - When i'm in insert mode, i try to type "jj" and it gets deleted and nvim exists INSERT mode and goes back to NORMAL mode. I need to prevent that from happening...

**Reply by siduck** [reply]
@noamso1 your jj is mapped, please remove it

**Reply by noamso1** [reply]
@siduck I can't find "jj" anywhere in ~/.config/nvim

**Reply by siduck** [reply]
@noamso1 check custom mappings

**Comment by noamso1** [comment]
@siduck Nothing there:
```
---@type MappingsTable
local M = {}

M.general = {
  n = {
    [";"] = { ":", "enter command mode", opts = { nowait = true } },
  },
  v = {
    [">"] = { ">gv", "indent"},
  },
}

M.nvimtree = {
  n = {
    ["<C-q>"] = { "<cmd> :qa<CR>" },
  },
}

-- more keybinds!

return M
```

**Reply by moisesmorillo** [reply]
@noamso1 check if you are using a plug in like this https://github.com/max397574/better-escape.nvim and disable it

**Comment by noamso1** [comment]
@moisesmorillo YES !! That's the bugger !! Thanks so much !

**Comment by noamso1** [comment]
Closing.. Resolved..
